src/main/kotlin/uk/gov/dluhc/printapi/config/CorrelationIdMdcConfiguration.kt

```diff
@@ -25,11 +33,14 @@ const val CORRELATION_ID_HEADER = "x-correlation-id"
  * HTTP header `x-correlation-id` if set. This allows for passing and logging a consistent correlation ID between
  * disparate systems or processes.
  */
```

Comment:
I know you didn't add this originally (I did 😁 ), but now we have the Web**Client** code below, I've started to get a bit confused by this interceptor method as to whether its client or server related.
I know its "server" related, as it's a **Handler**Interceptor .... but I had to think about it a bit ...

Do you think its worth adding to the kdoc to this method to make it clear that it's an interceptor for our request handler methods, and therefore is **server** related rather than client

Reply:
Added Kdoc so it's clear which one is for WebClient and which one is for RestControllers